pages/Ethereum/Identifiers.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Identifiers
+route: /Ethereum/Identifiers
+menu: "DSNP: Ethereum"
+---
+# Identifiers
+
+The [identity registry](/Ethereum/Registry) is the source of building and maintaining unique DSNP User Id.
```

Comment:
Missing "a" -- should be "building and maintaining **a** unique DSNP User Id", I assume.